FitGoal/FitGoal/ExerciseDetailViewController/ExerciseDetailViewController.swift

```diff
@@ -0,0 +1,111 @@
+//
+//  ExerciseDetailViewController.swift
+//  FitGoal
+//
+//  Created by Eliany Barrera on 13/3/20.
+//  Copyright © 2020 Eliany Barrera. All rights reserved.
+//
+
+import UIKit
+
+class ExerciseDetailViewController: UIViewController {
+    
+    var exercise: Exercise?
```

Comment:
If you pass the exercise in the initialiser then you won't have the optional type here.